commit 773ee752079f74032b5487bbc523adb901568e2f
Author: Heiko Becker <heirecka@exherbo.org>
Date:   Tue Jun 21 11:29:51 2016 +0200

    Pass -fext-numeric-literals to fix the build with Qt 5.7
    
    Otherwise I get "error: unable to find numeric literal operator
    'operator""Q' BOOST_DEFINE_MATH_CONSTANT(half, 5.000000000000000..."
    
    REVIEW: 128275

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8e2c4e8..5fec78a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,6 +70,7 @@ include_directories(
     ${Boost_INCLUDE_DIRS}
 )
 
+add_definitions(-fext-numeric-literals)
 add_definitions(-DQT_NO_CAST_TO_ASCII)
 add_definitions(-DQT_NO_URL_CAST_FROM_STRING)
 remove_definitions(-DQT_NO_CAST_FROM_ASCII)
